lib/active_support/cache/database_store.rb

```diff
@@ -21,6 +24,10 @@ def self.supports_cache_versioning?
       # option options [Class] :model model class. Default: ActiveSupport::Cache::DatabaseStore::Model
```

Comment:
Sorry, one thing I forgot to check. What I meant by "documentation" is adding a line here too, to document the new option. I would appreciate if could you do a quick fix PR?

Reply:
@dim https://github.com/bsm/activesupport-cache-database/pull/43